tests/suites/main_test.function

```diff
@@ -395,6 +395,7 @@ int main(int argc, const char *argv[])
                         {
                             /* Only one count is needed if not verbose */
                             unmet_dep_count++;
+                            unmet_dependencies[0] = NULL;
```

Comment:
If we want to make it more robust, then why not initialise all of them unconditionally in the beginning?

Reply:
Ok, might as well since the cost is negligible. (Filling `unmet_dependencies` unconditionally would be simpler, but it has a not-completely-negligible cost in heap size.)